### ISSUE TITLE
Upload artifact on maven central or github repository

### ISSUE THREAD

**Issue by kakawait** [beosign/snakeyaml-anno#26]
Do you plan to upload your lib directly to maven-central or at least to github maven repository?

It will help to use it by only putting line inside our `pom.xml`

**Comment by rvowles** [comment]
yes, i must admit it was disappointing find out i can't use this project.

**Comment by rvowles** [comment]
i suppose i can by forking it and giving it a pom and uploading it to central. I will make a minor readme change to make it clear it is your project however.

**Comment by marwin1991** [comment]
Hello, I use this lib by jitpack but maybe you want to upload it to central?
Here is a good step by step tutorial: https://medium.com/geekculture/how-to-publish-artifacts-on-maven-central-24342fd286cd
